sphinx_book_theme/_templates/sidebar-logo.html

```diff
@@ -1,7 +1,11 @@
 <div class="navbar-brand-box">
     <a class="navbar-brand text-wrap" href="{{ pathto('index') }}">
       {% if logo %}
-      <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
+        <!-- `logo` is deprecated in Sphinx 4.0, so remove this when we stop supporting 3 -->
```

Comment:
@choldgraf good find! Thanks.

Reply:
Oh cool it looks like there are `html_logo` and `latex_logo` now

https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=logo#confval-html_logo

This is something we will need to manage in jupyter-book to issue a depcration notice for `logo` in https://github.com/executablebooks/jupyter-book/issues/1438

Reply:
we may not need to deprecate it if we are managing it under the hood though!

Reply:
OK -- if `html` or `latex` isn't specified use `logo` -- I see where you are going